UPDATE: README page description (#42)
thoth-tech/documentation: README.md

## Contributing

Please see the [contribution guidelines](CONTRIBUTING.md).

## What is this repositry used for?

> This documentation repo is used to collate all information in regards to the Thoth-Tech Project. 

Reviewer: ```suggestion
This documentation repo is used to collate all information regarding Thoth Tech Products 
```

Author: Hi Talia, No worries at all. Thank you for your comments

Each team has their own separate repository and these are supposed to be used for anything related to the team projects. Additionally, if you have documentation elsewhere, we recommend that you begin a small migration process to this repository within a .md format.

Some of the things you might want to include within this repositry includes: 

* Meeting Minutes
* Research and Findings
* Onboarding Packages

> * And more - the potential is unlimited.

Reviewer: Using * as bullet points might be a cause of failing the formatting check

Consider 
- a dash for a list
